portal/templates/portal/mouseflow.html

```diff
@@ -0,0 +1,9 @@
+<script type="text/javascript">
+  var _mfq = _mfq || [];
+  (function() {
+    var mf = document.createElement("script");
+    mf.type = "text/javascript"; mf.async = true;
+    mf.src = "//cdn.mouseflow.com/projects/4e6407b5-890c-4dfc-9b50-fc14eeeeb2eb.js";
```

Comment:
It's client-side anyway, so everyone will be able to see this token, and I guess send fake data to mouseflow - like any client-side analytics